src/Illuminate/Cache/CacheManager.php

```diff
@@ -201,7 +201,7 @@ protected function createRedisDriver(array $config)
 	{
 		$redis = $this->app['redis'];
 
-		$connection = array_get($config, 'connection', 'default') ?: 'default';
+		$connection = array_get($config, 'connection', 'default');
```

Comment:
Just to point out the semantic difference here:
https://github.com/laravel/framework/blob/69877e7184757fc970e021248707167e7b7cf34c/src/Illuminate/Support/Arr.php#L215

If the value exists but is falsey, e.g. `''` or explicitly `null`, then the previous code would result in `'default'` and the new code would result in the falsey value.